docs/reference/signals.rst

```diff
@@ -0,0 +1,88 @@
+.. _signals:
+
+Signals
+=======
+
+Wagtail's `PageRevision <./pages/model_reference#pagerevision>`__  and
```

Comment:
if you link to the classes like this ``:class:`~wagtail.wagtailcore.models.PageRevision` ``. It'll generate the link to the reference section automatically.